README.md

```diff
@@ -0,0 +1,22 @@
+# WiseTech Global Analyzers
+
+[![Build](https://img.shields.io/appveyor/ci/WiseTechGlobal/wtg-analyzers/master.svg)](https://ci.appveyor.com/project/WiseTechGlobal/wtg-analyzersd)
```

Comment:
Both these appveyor links result in a `Project not found or access denied.` error. If they require special security rights to access, then we probably shouldn't link to them from the readme.